ams/server/src/main/java/com/netease/arctic/server/table/DefaultTableService.java

```diff
@@ -622,6 +623,11 @@ private boolean handleTableRuntimeAdded(
             tableIdentity.getDatabase(), tableIdentity.getTableName());
     AmoroTable<?> table =
         externalCatalog.loadTable(tableIdentity.getDatabase(), tableIdentity.getTableName());
+    if (TableFormat.ICEBERG == table.format()) {
```

Comment:
This would result in `table_identifier` having this table, but `table_runtime` and the `tableRuntimeMap` in memory do not have this table.

But I have checked the relevant codes, and it seems that it will not cause any issues. 👍